openpype/plugins/publish/extract_burnin.py

```diff
@@ -156,6 +155,11 @@ def main_process(self, instance):
             filled_anatomy = anatomy.format_all(burnin_data)
             burnin_data["anatomy"] = filled_anatomy.get_solved()
 
+            # Add context data burnin_data.
+            burnin_data["context"] = (
+                instance.context.data.get("burnin_context") or {}
```

Comment:
I'm still not very happy with the name. It's quite ambiguous. How about `custom_burnin_data` ?